samcli/local/init/templates/cookiecutter-aws-sam-hello-golang/{{cookiecutter.project_name}}/README.md

```diff
@@ -77,38 +77,21 @@ AWS Lambda Python runtime requires a flat folder with all dependencies including
             ...
 ```
 
-First and foremost, we need a `S3 bucket` where we can upload our Lambda functions packaged as ZIP before we deploy anything - If you don't have a S3 bucket to store code artifacts then this is a good time to create one:
+To deploy your application for the first time, run the following in your shell:
 
 ```bash
-aws s3 mb s3://BUCKET_NAME
+sam deploy --guided
```

Comment:
Do we need `sam build` here?

Reply:
No, Go build is not yet supported. Once it is, we would add it there.